homeassistant/components/alert.py

```diff
@@ -249,12 +255,22 @@ def _notify(self, *args):
 
         if not self._ack:
             _LOGGER.info("Alerting: %s", self._name)
+            self._send_done_message = True
             for target in self._notifiers:
                 yield from self.hass.services.async_call(
                     'notify', target, {'message': self._name})
         yield from self._schedule_notify()
 
     @asyncio.coroutine
+    def _notify_done_message(self, *args):
+        """Send notification of complete alert."""
+        _LOGGER.info("Alerting: %s", self._done_message)
+        self._send_done_message = True
```

Comment:
It looks like this should be set to `False` instead of `True` ?

Maybe add a test to make sure the bug won't happen ;-)